packages/scandipwa/src/component/Breadcrumbs/Breadcrumbs.component.js

```diff
@@ -48,7 +48,8 @@ export class Breadcrumbs extends PureComponent {
     renderBreadcrumbList(breadcrumbs) {
         const breadcrumbsWithHome = [
             ...breadcrumbs,
-            { url: '/', name: __('Home') }
+            // Looks like a browser bug, temporary fixed with .toLowerCase()
+            { url: '/', name: __('Home').toLowerCase() }
```

Comment:
Please figure this out. Very weird! Let's omit any crotches in our applications.

Reply:
Temporarily fixed as agreed

Reply:
FYI this is probably related to the i18n rendering each letter as a separate node.